packages/documentation-spa/docs/apps/internal/lighthouse/lighthouse-spa.md

```diff
@@ -0,0 +1,93 @@
+---
+id: lighthouse-spa
+title: Lighthouse
+sidebar_label: Lighthouse
+slug: /apps/internal/lighthouse
+---
+* * *
+
+## Developers
+
+### Component Contributors
+
```

Comment:
```suggestion

```

Reply:
Changed thanks for pointing out. We need to build a  standard skeleton for documentation.